src/Util/Api.php

```diff
@@ -95,6 +98,30 @@ public function getLang(): string {
 		return $this->lang;
 	}
 
+	/**
+	 * Get the on-wiki configuration from the `MediaWiki:WS_Export.json` page. Cached for 1 month.
+	 * @return mixed[] With keys: defaultFont
+	 */
+	public function getWikiConfig(): array {
+		if ( isset( $this->wikiConfig[ $this->getLang() ] ) ) {
+			return $this->wikiConfig[ $this->getLang() ];
+		}
+		$this->wikiConfig[ $this->getLang() ] = $this->cache->get( 'wikiConfig_' . $this->getLang(), function ( CacheItemInterface $cacheItem ) {
+			$cacheItem->expiresAfter( new DateInterval( 'P1M' ) );
+			$configPageName = 'MediaWiki:WS_Export.json';
+			$json = $this->get( 'https://' . $this->getDomainName() . '/w/index.php?title=' . $configPageName . '&action=raw&ctype=application/json' );
+			$data = json_decode( $json, true );
+			if ( $data === null ) {
+				return [];
```

Comment:
Are we OK breaking backward compatibility by removing the defaults for non-latin languages? Should we provide some defaults?

Reply:
I mentioned it in the commit message: we'll migrate these defaults to the on-wiki configs on the respective wikis. There might be a short time when things don't work correctly, but not long.